### ISSUE TITLE
Ability to change the main theme colors

### ISSUE THREAD

**Issue by wdanilo** [jmcalaway/atom-alphabet-ui#3]
Hello!
Woud it be possible to add possibility to change the following theme colors?
1. sidebar background
2. sidebar dimmed font
3. sidebar undimmed font
4. sidebar open file font (when the file is opened and undimmed, there is no color difference like it was in the Genesis theme)

Moreover I would beg you to remove the white "addons" - they are simply burning my eyes when working long in the night. Is there any way to change the colors by end-user without making a branch from your theme?

**Comment by nick5435** [comment]
+1 Would like to see this.

**Comment by skbolton** [comment]
Yeah I would like to see this too. One of my favorite parts of atom material ui is the fact that it can adapt to any syntax theme. I want to use Climate UI but I hate the color clash when I use it with other syntaxs.
